Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.6-alpine
+
+COPY requirements.txt /app/
+
+RUN true \
+	&& pip install gunicorn \
+	&& pip install -r /app/requirements.txt \
+	&& true
+
+WORKDIR /app/src
```

Comment:
The decision of the project folder structure should be made on a mentoring session (so others can learn why it is good to do it in a particular way). If we decide there that it is better to have another directory structure, I (or somebody else) will move it accordingly to the decision.